<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qCjyxVolumesModuleWidget</class>
 <widget class="qCjyxWidget" name="qCjyxVolumesModuleWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>247</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Volumes</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Icons/Volumes.png</normaloff>:/Icons/Volumes.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="ActiveVolumeLabel">
       <property name="text">
        <string>Active Volume: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="qDMMLNodeComboBox" name="ActiveVolumeNodeSelector">
       <property name="nodeTypes">
        <stringlist>
         <string>vtkDMMLVolumeNode</string>
        </stringlist>
       </property>
       <property name="addEnabled">
        <bool>false</bool>
       </property>
       <property name="renameEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="InfoCollapsibleButton">
     <property name="text">
      <string>Volume Information</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="qDMMLVolumeInfoWidget" name="DMMLVolumeInfoWidget"/>
      </item>
      <item>
       <widget class="QFrame" name="ConvertVolumeFrame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="ConvertVolumeLabel">
           <property name="text">
            <string>Convert to LabelMap:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>32</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="qDMMLNodeComboBox" name="ConvertVolumeTargetSelector">
           <property name="nodeTypes">
            <stringlist>
             <string>vtkDMMLLabelMapVolumeNode</string>
            </stringlist>
           </property>
           <property name="noneEnabled">
            <bool>true</bool>
           </property>
           <property name="renameEnabled">
            <bool>true</bool>
           </property>
           <property name="noneDisplay">
            <string>Replace current node</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ConvertVolumeButton">
           <property name="text">
            <string>Convert</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="DisplayCollapsibleButton">
     <property name="text">
      <string>Display</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="qCjyxVolumeDisplayWidget" name="VolumeDisplayWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="ColorLegendCollapsibleButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Color Legend</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="qDMMLColorLegendDisplayNodeWidget" name="ColorLegendDisplayNodeWidget">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qDMMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qDMMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qDMMLVolumeInfoWidget</class>
   <extends>qDMMLWidget</extends>
   <header>qDMMLVolumeInfoWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qDMMLWidget</class>
   <extends>QWidget</extends>
   <header>qDMMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qCjyxWidget</class>
   <extends>QWidget</extends>
   <header>qCjyxWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qDMMLColorLegendDisplayNodeWidget</class>
   <extends>qDMMLWidget</extends>
   <header>qDMMLColorLegendDisplayNodeWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qCjyxVolumeDisplayWidget</class>
   <extends>qDMMLWidget</extends>
   <header>qCjyxVolumeDisplayWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qCjyxVolumesModule.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>qCjyxVolumesModuleWidget</sender>
   <signal>dmmlSceneChanged(vtkDMMLScene*)</signal>
   <receiver>DMMLVolumeInfoWidget</receiver>
   <slot>setDMMLScene(vtkDMMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qCjyxVolumesModuleWidget</sender>
   <signal>dmmlSceneChanged(vtkDMMLScene*)</signal>
   <receiver>ActiveVolumeNodeSelector</receiver>
   <slot>setDMMLScene(vtkDMMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>1</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qCjyxVolumesModuleWidget</sender>
   <signal>dmmlSceneChanged(vtkDMMLScene*)</signal>
   <receiver>ConvertVolumeTargetSelector</receiver>
   <slot>setDMMLScene(vtkDMMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>568</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>499</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qCjyxVolumesModuleWidget</sender>
   <signal>dmmlSceneChanged(vtkDMMLScene*)</signal>
   <receiver>ColorLegendDisplayNodeWidget</receiver>
   <slot>setDMMLScene(vtkDMMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>413</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
